modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,7 +11,10 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support NodePort services.
+{product-title} with Kuryr SDN does not support NodePort services. Also, if
+the Machines Subnet is not connected to a Router or it's, but the Router is
```

Comment:
What is "or it's" here?

Reply:
I meant "or it's connected". Let me rephrase it.